case/interactive/T0000_rackhd_OSInstall.py

```diff
@@ -114,13 +117,21 @@ def test(self):
             return
 
         # Start workflow to install OS
+        node_workflow_dic = {}
         for node_id, node in node_os_install_list:
             node_bmc_ip = node.get_bmc_ip()
             self.monorail.put_node_ipmi_obm(
                 node_id=node_id, host=node_bmc_ip, username="admin", password="admin")
             node.install_os(self.data["os_name"])
             self.log(
                 'WARNING', 'Installation on node {} starts, please open VNC to check'.format(node_id))
+            rsp = node.get_workflows(active=True).items()
+            if len(rsp) != 0:
+                workflow_obj = rsp[0][1]
+                workflow_instance_id = workflow_obj.instanceId
+                node_workflow_dic[node_id] = workflow_instance_id
```

Comment:
A good manner is to check OS install is in progress, so the code `if "Install" in workflow_obj.injectableName:` can be used here as well.

Reply:
I can do that.

Reply:
Included in the last PR.